README.md

```diff
@@ -2,3 +2,171 @@
+services/users $ go mod tidy
+go: finding module for package github.com/datadog/orchestrion/instrument
+go: found github.com/datadog/orchestrion/instrument in github.com/datadog/orchestrion v0.1.0
+services/users $ cd ../notes/
+services/notes $ go mod tidy
+go: finding module for package github.com/datadog/orchestrion/instrument
+go: found github.com/datadog/orchestrion/instrument in github.com/datadog/orchestrion v0.1.0
+services/notes $ cd ../../
+$ 
+```
+
+Then rebuild the applications with our newly instrumented code
+```sh 
+$ docker-compose build
+```
+
+And start the services again:
+| :zap: Remember that the DD_API_KEY environment variable needs to be set! |
+|--------------------------------------------------------------------------|
+```sh 
+$ docker-compose up -d
+```
+
+Now, going to the application home page [http://localhost:8080](http://localhost:8080) will show us the same page, but looking at our Traces Page in the Datadog app [https://app.datadoghq.com/apm/traces](https://app.datadoghq.com/apm/traces) should show some new traces appear!
+
+![First Trace](doc/trace1.png)
+
+Click through the various spans and tabs. For instance, by clicking the `sqlite3.query` span, we can see the SQL query and other info.
+![SQLite3 Trace Data](doc/sqlite-info.png)
+
+
+## More Traces
+Now that we have instrumentation, let's generate some more interesting traces. From the browser, click on one of the users' IDs to go to their notes page.
+
+Navigating to that page should generate another trace, which looks like this:
+![Notes Page](doc/notes-trace.png)
+
+Here we can see that the `users` service received an http request. It then sent an outgoing http request to the `notes` service, requesting `/notes?userid=2` in this case (this will depend on which user's page you are viewing).
+![Outgoing](doc/notes-trace2.png)
+
+The `notes` service then executes a SQL query:
+![Notes SQL Query](doc/notes-sql.png)
+
+
+Try adding a new note. When you do that, you should see a new trace appear.
+![New Note Trace](doc/notes-trace3.png)
+This trace appears with type `POST` and status code `302` because the `users` service returns a redirect to the browser after the submission succeeds.
+
+Once again, we can see in the trace that the `users` service talks to the `notes` service, this time resulting in an `INSERT` query.
+![New Note Trace Details](doc/notes-trace4.png)
+
+## Removing Instrumentation
+Now that we've seen how to add instrumentation to the code, we might as well cover how to remove it. 
+
+Removing instrumentation is as easy as adding it. If, for whatever reason, you want to remove the instrumentation added by `orchestrion`, run the following in the repository root:
+```sh 
+$ orchestrion -rm -w .
+Scanning Package /.../go-sample-app
+Removing Orchestrion instrumentation.
+overwriting /.../go-sample-app/services/notes/main.go:
+overwriting /.../go-sample-app/services/users/main.go:
+overwriting /.../go-sample-app/tools/header_check.go:
+```
```

Comment:
````suggestion
```shell
$ orchestrion -rm -w .
```
````
Similar to the one of the previous suggestions, it think it's simpler to only list the commands a user needs to execute.